src/engine/Match_tainting_mode.mli

```diff
@@ -58,6 +58,13 @@ val taint_config_of_rule :
   unit) ->
   Dataflow_tainting.config * debug_taint * Matching_explanation.t list
 
+val mk_params_env :
```

Comment:
maybe a small comment before explaining the goal of the function and why it's exposed?